docs/launching-the-exploit.md

```diff
@@ -2,17 +2,15 @@
 title: Launching the Exploit
 ---
 
-We'll start by first setting up your SD card to launch our exploit. For most users, we recommend using an exploit called "Memory Pit" which takes advantage of a flaw in how the Nintendo DSi Camera application handles image metadata.
+We'll start by first setting up your SD card to launch our exploit. For most users, we recommend using an exploit called "Memory Pit" which takes advantage of a flaw in how the Nintendo DSi Camera application handles image metadata.     
+If you don't plan on installing Unlaunch, it is recommended to use [Flipnote Lenny](lauching-the-flipnote-exploit) instead. ([Why?](faq.html#why-use-flipnote-lenny-over-memory-pit))
```

Comment:
Might be good to either link the unlaunch benefits here or change that FAQ to "Which is the best exploit for me?" or so with the benefits of all three.